packages/coding-agent/src/clipboard.ts

```diff
@@ -0,0 +1,28 @@
+import { execSync } from "child_process";
+import { platform } from "os";
+
+export function copyToClipboard(text: string): void {
+	const p = platform();
+	const options = { input: text, timeout: 5000 };
+
+	try {
+		if (p === "darwin") {
+			execSync("pbcopy", options);
+		} else if (p === "win32") {
+			execSync("clip", options);
+		} else {
+			// Linux - try xclip first, fall back to xsel
+			try {
+				execSync("xclip -selection clipboard", options);
```

Comment:
Consider adding the `-i` flag: `xclip -selection clipboard -i`. Some xclip versions require it to explicitly read from stdin.

(Reviewed via playwriter.dev)